labellab-flask/api/serializers/image.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import Schema, fields
+
+from api.extensions import db, ma
+from api.serializers.labeldata import LabelDataSchema
+
+class ImageSchema(ma.ModelSchema):
+    """
+    Serializer class for image
+    """
+        
+    id = fields.Int(dump_only=True) 
+    imagename = fields.Str()
```

Comment:
it should be image_name

Reply:
Sure but for that I will also have to change the names in the db